reference/latest/src/main/java/com/example/docs/item/armor/GuiditeArmorMaterial.java

```diff
@@ -0,0 +1,107 @@
+package com.example.docs.item.armor;
+
+import net.minecraft.item.ArmorItem;
+import net.minecraft.item.ArmorMaterial;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.sound.SoundEvents;
+
+import com.example.docs.item.ModItems;
+
+// :::1
+public class GuiditeArmorMaterial implements ArmorMaterial {
+	//:::1
+	// :::_10
+	public static final GuiditeArmorMaterial INSTANCE = new GuiditeArmorMaterial();
+	// :::_10
+	// :::1
+	// Base durability values for all the slots.
+	// Boots, Leggings, Chestplate, Helmet
+	// You shouldn't change these values.
+	private static final int[] BASE_DURABILITY = new int[] {13, 15, 16, 11};
```

Comment:
These 2 int arrays are quite confusing, why can this one not just have the durability values that are going to be used?

WHy does the other have constants for the 3 entries, but this one doesnst?